src/sagas/driveTimerSaga.js

```diff
@@ -0,0 +1,19 @@
+import { call, put, takeLatest } from 'redux-saga/effects';
+
+import { getDriveTimer } from 'apis/driveTimerApi';
+import { driveTimerAction, driveTimerRequestFailed } from 'actions/driveTimerActions';
+
+import { DRIVE_TIMER } from 'constants/actionConstants';
+
+export function* driveTimer() {
```

Comment:
Codacy has a fix for the issue: [Missing space before *.](https://app.codacy.com/gh/joshsoftware/OnCoT-Frontend/pullRequest?prid=7073195)
```suggestion
export function * driveTimer() {
```